docs/en-branded/revisions.md

```diff
@@ -1,3 +1,4 @@
 | Date          | Change            |
 |---------------|-------------------|
-| January 2022 | Intitial release of version 1.0.0 |
\ No newline at end of file
+| January 2022 | Intitial release of version 1.0.0 |
+| December 2022 | Released version 1.1.0 <br> 1. Support Arabic and Portuguese languages.<br> 2. Support custom vocabulary.<br> 3. Reduce noise to improve captions quality<br> 4. Reduce captions translation latency<br> |
```

Comment:
Support Arabic and Portuguese for the default caption processing language.

Reply:
Fixed